jax/_src/api.py

```diff
@@ -2814,6 +2814,25 @@ def eval_shape(fun, *args, **kwargs):
   (2000, 1000)
   >>> print(out.dtype)
   float32
+
+  In case the function has static arguments, it should be wrapped
+  using :func:`functools.partial` before passing to :func:`eval_shape`:
+
+  >>> import jax
+  >>> from jax import lax
+  >>> from functools import partial
+  >>> import jax.numpy as jnp
```

Comment:
Explicit imports of `jax`, `lax`, and `jnp` are not necessary in doctests because they are imported automatically in the doctest environment

Reply:
From a users perspective I must admit it is very convenient to just copy and paste code from docs and being able to just run it. The other, first example of `eval_shape` also uses explicit imports, but I'm happy to adopt to whatever makes it consistent.